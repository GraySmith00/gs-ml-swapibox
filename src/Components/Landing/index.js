import React from 'react';

const Landing = props => {
  return <div />;
};

export default Landing;
