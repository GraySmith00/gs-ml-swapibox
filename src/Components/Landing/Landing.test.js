import React from 'react';
import { shallow } from 'enzyme';
import Landing from '.';

describe('Landing component', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Landing />);
  });

  it('should match the snapshot', () => {
    expect(wrapper.html()).toMatchSnapshot();
  });
});
