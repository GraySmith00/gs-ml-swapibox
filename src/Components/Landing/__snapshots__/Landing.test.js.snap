// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Landing component should match the snapshot 1`] = `"<div><h1>Landing</h1></div>"`;
